src/Microsoft.OData.Core/Microsoft.OData.Core.csproj

```diff
@@ -92,9 +92,6 @@
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces">

```

Comment:
is it a breaking changes to downgrade the dependency version? 

Why can't we upgrade the dependency version in the test project?